### PR TITLE
feat(web-app): setup vscode-apollo and init graphql apollo client

### LINKED ISSUE #48: Setup Apollo GraphQL IDE plugin
Follow [this guide](https://www.apollographql.com/docs/platform/editor-plugins.html)

### PR DESCRIPTION
**Pull request by nguyenthienthanh**
NOTE: Please use **vscode-apollo** version 1.5.0 to resolve this issue [48#476509724](https://github.com/targeek/guru-erp/issues/48#issuecomment-476509724)
Resolve #48

### CODE REVIEW

**Comment by nguyenthienthanh** [comment]
[Issue 48: Setup Apollo GraphQL IDE plugin](https://trello.com/c/LNqrnpQX/4-issue-48-setup-apollo-graphql-ide-plugin)

File: packages/web-app/.env [hunk 1]
```diff
@@ -1 +1,2 @@
 NODE_PATH=src
+SERVER_PATH=http://localhost:4000
```
Comment (duongdev) [inline packages/web-app/.env]: This should be added on `.env.development` instead of `.env`

File: packages/web-app/package.json [hunk 2]
```diff
@@ -13,6 +13,9 @@
     "@material-ui/core": "4.0.0-alpha.7",
     "@material-ui/lab": "4.0.0-alpha.6",
     "@material-ui/styles": "4.0.0-alpha.7",
+    "apollo-cache-inmemory": "^1.5.1",
```
Comment (duongdev) [inline packages/web-app/package.json]: We currently don't need these deps in production. Pls remove

File: packages/web-app/src/graphql/client.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import unfetch from 'unfetch'
```
Comment (duongdev) [inline packages/web-app/src/graphql/client.ts]: Remove this file too

**Pull request closed without merging**